explainer-server/public/javascripts/explain-editor-plain-js.js

```diff
@@ -43,13 +43,26 @@ function updateCheckboxState() {
  * Tag Search
  */
 
-function processCapiSearchResponse(response){
+
+function processCapiSearchResponseTags(divIdentifier,response,userInterfaceTagDescriptionKey){
+    response.results.forEach(function(tag){
```

Comment:
I actually prefer taking time understanding how to correctly convert the JS values to Scala values (much more useful skill) than introducing a dependency.